client/packages/core/src/Reactor.js

```diff
@@ -365,7 +365,7 @@ export default class Reactor {
   _reactorStats() {
     return {
       inFlightMutationCount: this._inFlightMutationEventIds.size,
-      pendingMutationCount: this._pendingMutations().size,
+      storedMutationCount: this._pendingMutations().size,
```

Comment:
This was confusing people in the chat--they thought it meant the number of mutations that hadn't been applied by the server, but we'll keep mutations around for a while since we don't know if a query is waiting for a refresh.